crt_portal/static/sass/custom/buttons.scss

```diff
@@ -40,9 +40,4 @@ button#login {
     position: relative; // won't change display of card but will allow absolute positioning of button
     padding-bottom: units($spacer-6x); // to accommodate button
   }
-  button[name='wizard_goto_step'] {
```

Comment:
@hbillings I had to remove this due to issues on mobile -- not entirely sure why we had this added, I'm sure I'm missing something here.

Reply:
ooooh, my bad! That was what moved the edit buttons on the review page cards to the bottom of the card. Clearly I should have given them a class name instead of targeting the `name` attribute! I can add it back in with a classname.

Reply:
Done in #536